pkg/registry/core/adapters/nse_context_client_test.go

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) 2020 Doc.ai and/or its affiliates.
```

Comment:
Also, let's put these tests to adapters/nse_registry_test.go

Reply:
ok, done